packages/shared/jest-config/lib/jest.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   preset: 'ts-jest',
   projects: ['<rootDir>/packages/**/jest.config.js'],
   testEnvironment: 'node',
-  testRegex: ['(.*)(\w+)\/__tests__\/(.*).spec.ts'],
```

Comment:
Not allow run the tests in files that not has the extension `.spec.ts`  and not have are inside folder `__tests__`